includes/class-amp-theme-support.php

```diff
@@ -1179,8 +1180,9 @@ public static function register_content_embed_handlers() {
 				);
 				continue;
 			}
-
-			$embed_handler->register_embed();
+			if ( $embed_handler instanceof Registerable ) {
```

Comment:
Will this have a backwards-incompatibility problem? If someone has existing subclasses of `AMP_Base_Embed_Handler` with `register_embed` defined, should it not do something like this:

```suggestion
			if ( $embed_handler instanceof Registerable || method_exists( $embed_handler, 'register_embed' ) ) {
```